docs/release_notes.rst

```diff
@@ -93,6 +93,12 @@ inputs on other systems.
 [RTI Issue ID CON-190]
 
 
+Assessments related to vulnerabilities
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+RTI has assessed the current version of Lua used by *Connector*, version 5.2, 
```

Comment:
I do not know if most users of Connector will be aware that Lua is used internally. Maybe we should preface this sentence with something like "Internally, *Connector* relies on Lua." or similar.

what do you think?

Reply:
That makes sense.

Reply:
Also, per Jose, we are using this wording: 
RTI has assessed the current version of Lua used by *Connector*, version 5.2., which is is currently not affected by publicly disclosed vulnerabilities.